### ISSUE TITLE
Any assignment at the root of a SFC not supported

### ISSUE THREAD

**Issue by thomasdelobel** [vue-styleguidist/vue-styleguidist#182]
Hello !

First, thank you all for this great job :)

It seems that object destructuring (or maybe even any assignation) made at the root of the  `<script>` section is not recognized by the parser and make it crashes.

When a SFC contains something like this :

```
<template>
    ...
</template>

<script>
    ...
    const { HIGH, NORMAL } = AppTypes.FlagLevel
    <<OR>>
    const LOGS_ENABLED = config.logs.router
    ...
    export default {
        name: 'FlagBanner',
        computed: {}
        methods: {}
        ...
    }
</script>
```

This kind of warning is raised when compiling :

```
Warning: Cannot parse src/components/app/FlagBanner/FlagBanner.vue: TypeError: Cannot read property 'HIGH' of undefined

It usually means that vue-docgen-api does not understand your source code or when using third-party libraries, try to file an issue here:
https://github.com/vue-styleguidist/vue-docgen-api/issues
```

I tried in vain to find out where this issue could come from. Do you have any idea about where it actually crashes and how we could fix this ?

**Comment by rafaesc** [comment]
It is related with https://github.com/vue-styleguidist/vue-docgen-api/issues/23#issuecomment-371202215

**Comment by elevatebart** [comment]
@thomasdelobel this issue seems to be obsolete, if you encounter this bug in 3.X please open a new one

Thank you.
